modules/meteroid/crates/diesel-models/src/plan_versions.rs

```diff
@@ -22,7 +22,7 @@ pub struct PlanVersionRow {
     pub billing_cycles: Option<i32>,
     pub created_at: NaiveDateTime,
     pub created_by: Uuid,
-    pub billing_periods: Vec<BillingPeriodEnum>,
+    pub billing_periods: Vec<Option<BillingPeriodEnum>>,
```

Comment:
what is the point in having `Vec<Option<...>>`? I suspect it was added as diesel generates those Array<Nullable<...>>?